src/main/java/org/jabref/logic/crawler/StudyRepository.java

```diff
@@ -280,7 +279,7 @@ private String findQueryIDByQueryString(String query) {
      * @param crawlResults The results that shall be persisted.
      */
     private void persistResults(List<QueryResult> crawlResults) throws IOException {
-        DatabaseMerger merger = new DatabaseMerger(keywordDelimiter);
+        DatabaseMerger merger = new DatabaseMerger(importFormatPreferences.getKeywordSeparator());
```

Comment:
The keyword separator and delimiter are different the delimiter is groups related

Reply:
Oh ok, I see. Think I was a little bit too sloppy here...

Reply:
Ok, I checked it, it's a bit confusing, but I think, this is correct here:
`importFormatPreferences.getKeywordSeparator()` returns `prefs.getKeywordDelimiter()`, which is by default `,`. In this case, `keywordDelimiter` was originally initialized with an argument in the constructor in `StudyRepository.java`, which is called in `StartLiteratureReviewAction.java`, which is called by `JabRefFrame.java` with the argument `prefs.getKeywordDelimiter()`. As `prefs.getImportExportPreferences().getKeywordSeparator()` returns the same as `prefs.getKeywordDelimiter(), this should be right here: can you please doublecheck this?

Reply:
Ah yeah, it's indeed the same. I overlooked that it's the same as in the ImportFormatPreferences, I thought it were two different ones.